mvn_setup/tasks-parent/tasks-ui/src/core/Assignee/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { AvatarGroup, Box, ListItemText, InputAdornment, ListItem, Checkbox, Button, Avatar, List, TextField, ButtonProps, styled, ListItemTextProps } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+
+import Client from '@taskclient';
+
+import { TaskDescriptor } from 'taskclient/tasks-ctx-types';
+import { usePopover } from 'core/TaskTable/CellPopover';
+import { useAssignees } from 'taskclient/hooks';
+
+
+export const StyledButton = styled(Button)<ButtonProps>(() => ({
+  variant: 'text',
+  color: 'inherit',
+  "&.MuiButtonBase-root": {
+    minWidth: "unset",
+  },
+}));
+
+export const StyledListItemText = styled(ListItemText)<ListItemTextProps>(({theme}) => ({
+  marginLeft: theme.spacing(1)
+}));
+
+const SearchField: React.FC<{ onChange: (value: string) => void }> = ({ onChange }) => {
+  return (
+    <TextField
+      InputProps={{
+        startAdornment: (
+          <InputAdornment position="start">
+            <SearchIcon color='primary' />
+          </InputAdornment>
+        ),
+      }}
+      fullWidth
+      variant='standard'
+      placeholder='Search'
+      onChange={(e) => onChange(e.target.value)}
+    />
+  );
+}
+
+const UserAvatar: React.FC<{ children?: Client.AvatarCode, onClick?: (event: React.MouseEvent<HTMLElement>) => void }> = ({ children, onClick }) => {
+  const { state } = Client.useTasks();
+  const bgcolor: string | undefined = children ? state.pallette.owners[children.value] : undefined;
+
+  return (<Avatar onClick={onClick}
+    sx={{
+      bgcolor,
+      width: 24,
+      height: 24,
+      fontSize: 10,
+    }}>
+
+    {children ? children.twoletters : <PersonAddIcon sx={{ fontSize: 15 }} />}
+  </Avatar>
+  );
+}
+  
+const Assignee: React.FC<{ task: TaskDescriptor }> = ({ task }) => {
```

Comment:
This is not "Assignee". You have a dependency on a task in here, so by nature, it can't just be any Assignee.  This is TaskAssignees (plural) because a Task can have more than one Assignee.  Rename the constant and the Folder to TaskAssignees.  

It is important to differentiate here, because we will most likely have multiple other types of Assignees, like ChecklistAssignees and CommentAssignees, etc.